src/condor_tests/test_job_submit_method_recording.py

```diff
@@ -7,21 +7,35 @@
 #them for evaluation.
 #
 #Written by: Cole Bollig @ March 2022
+#
+#Tickets:
+# Base: HTCONDOR-996
 
 from ornithology import *
 import os
 from time import sleep
 from time import time
 
-
+#-----------------------------------------------------------------------------------------
 #Function to wait until schedd has no jobs
 def wait(schedd):
      t_end = time() + 180 #Give a 2 minute maximum for waiting
      while any(schedd.query(constraint='true',projection=["JobSubmitMethod"])) == True:
           sleep(0.1)
           if time() >= t_end:
+               print("Job took longer than 2 minutes to complete")
                break
-
+#-----------------------------------------------------------------------------------------
+#Function to clean up files created by dags
+def clean_up(default_condor, t_dir, dir_name):
+     #List of files needed to be moved
+     files_to_move = ["simple.dag.condor.sub","simple.dag.dagman.log","simple.dag.dagman.out","simple.dag.lib.err","simple.dag.lib.out","simple.dag.metrics","simple.dag.nodes.log"]
+     #make directory dag submission tests to avoid errors
+     p = default_condor.run_command(["mkdir",t_dir / "dag_test_{0}".format(dir_name)])
```

Comment:
Let's try to do thing like mkdir in native python calls.  These are likely to be more portable (e.g. to Windows), and run faster.